packages/js/src/utils/angular.ts

```diff
@@ -46,6 +46,11 @@ export type LogtoAngularConfig = {
    * @default Prompt.Consent
    */
   prompt?: Prompt | Prompt[];
+  /**
+   * The routes protected by the `angular-auth-oidc-client` library.
+   * The access token will be added automatically to the `Authorization` header for these routes.
+   */
+  secureRoutes?: string[];
```

Comment:
This function was intended to transpile the options into a "Logto" way. But this one is the option from the Angular library. We can use object spreading to merge configs instead of adding a new key for this function?

Reply:
Yeah, that makes sense

Reply:
Code updated to allow any `OpenIdConfiguration` config to be indicated in `buildAngularAuthConfig` util, which would be spreaded and returned as-is.